examples/ecs/canary/.pipe.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: pipecd.dev/v1beta1
+kind: ECSApp
+spec:
+  input:
+    serviceDefinitionFile: servicedef.yaml
+    taskDefinitionFile: taskdef.yaml
+    targetGroups:
+      # Primary target group is pointed by LB (ALB/NLB) live listener, the Primary task set
+      # will be registered to this target group.
+      primary:
+        targetGroupArn: arn:aws:elasticloadbalancing:ap-northeast-1:XXXX:targetgroup/ecs-canary-blue/YYYY
+        containerName: web
+        containerPort: 80
+      # Canary target group can also be pointed by LB (ALB/NLB) live listener at
+      # the beginning of deployment, the Canary task set will be registered to this target group.
+      # CAUTION: To enable PipeCD performs canary deployment strategy, this canary target group
+      # is required.
+      canary:
+        targetGroupArn: arn:aws:elasticloadbalancing:ap-northeast-1:XXXX:targetgroup/ecs-canary-green/ZZZZ
+        containerName: web
+        containerPort: 80
+  pipeline:
+    stages:
+      # Rollout CANARY variant's workload.
+      # scale represents the percentage of workload will be rolled out,
+      # in this case 10% of the current PRIMARY variant's workload is set.
+      # Note: Rolled out variant serves no traffic at this time.
+      - name: ECS_CANARY_ROLLOUT
+        with:
+          scale: 10
```

Comment:
10% of workloads will receive 20% of traffic is not a best practice so I think we can change this to 30%.

Reply:
Nice catch ⚾

Reply:
addressed on https://github.com/pipe-cd/pipe/pull/2079/commits/fd360ed6ee4989ef6e41ce9e71630531b7c3ac8b 🙏